app/(auth)/account.tsx

```diff
@@ -18,6 +21,11 @@ export default function AccountPage() {
     getProfile: state.getProfile,
   }));
 
+  const handleSignOut = async () => {
```

Comment:
This function was missing, restoring it should restore expected behavior. Currently this is navigation to the sign-up page, is that OK or should it go to a splash page?